src/HelpText.cpp

```diff
@@ -411,9 +411,9 @@ const URLString VerCheckUrl()
 // const URLString VerCheckUrl()
 // {
 //    //The version we intend to use for live Sneedacity.
-// #define VER_CHECK_URL "https://www.sneedacityteam.org/download/?"
+// #define VER_CHECK_URL "https://www.sneedacity.org/download/?"
```

Comment:
Doesn't currently work, but if we get a stable release going, we can use this

Reply:
Is that for retrieving update notifications from the net? If so then it needs to be made opt-in.

Reply:
That can be done in another PR. Right now I'm focusing on fixing links.

Reply:
possibly dead code at this point since the entirety of the network and curl stuff is out

Reply:
And it's already commented out, so it shouldn't phone home anyway.